pkg/kubelet/kubelet_network_linux.go

```diff
@@ -47,14 +47,16 @@ func (kl *Kubelet) initNetworkUtil() {
 		iptClients = append(iptClients, utiliptables.New(exec, utiliptables.ProtocolIPv6))
 	}
 
-	for _, iptClient := range iptClients {
-		kl.syncNetworkUtil(iptClient)
-		go iptClient.Monitor(
-			utiliptables.Chain("KUBE-KUBELET-CANARY"),
-			[]utiliptables.Table{utiliptables.TableMangle, utiliptables.TableNAT, utiliptables.TableFilter},
-			func() { kl.syncNetworkUtil(iptClient) },
-			1*time.Minute, wait.NeverStop,
-		)
+	for i := range iptClients {
+		iptClient := iptClients[i]
+		if kl.syncNetworkUtil(iptClient) {
+			go iptClient.Monitor(
```

Comment:
Since this only runs once, we can log to inform that the syncNetwork was succesfully on ipfamily X

Reply:
or maybe add it to the `syncNetworkUtil()` before returning?